project/src/main/java/com/google/sps/servlets/EventVolunteeringDataServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
```

Comment:
And this file should be in another PR. So every PR can be small, in order to be reviewed fast and merged fast.

Reply:
Done - Yes, I think it is organized better now, thanks.